docs_user/modules/proc_rolling-back-the-control-plane-adoption.adoc

```diff
@@ -2,27 +2,19 @@
 
 = Rolling back the control plane adoption
 
-If you encountered a problem during the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services that prevents you from completing the adoption procedure, you can roll back the control plane adoption.
+If you encountered a problem that prevents you from completing the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services, you can roll back the control plane adoption.
 
 [IMPORTANT]
-The roll back operation is only possible during the
-control plane parts of the adoption procedure. If you altered the data plane
-nodes in any way during the procedure, the roll back is not possible.
-
-During the control plane adoption, services on the source cloud's
-control plane are stopped but not removed. The databases on the source
-control plane are not edited by the adoption procedure. The
-destination control plane received a copy of the original
-control plane databases. The roll back procedure assumes that the data
-plane has not yet been touched by the adoption procedure and it is
-still connected to the source control plane.
+You can only roll back the control plane adoption during the parts of the adoption procedure that alter the control plane. If you altered the data plane
```

Comment:
Maybe:
"If you altered the data plane nodes in any way, you cannot perform the rollback procedure. You can only roll back the control plane adoption when the control plane is altered. "

(I might put the kind of warning first as that seems more important to me)